### ISSUE TITLE
What does connectable mean in the output of "vlab status"?

### ISSUE THREAD

**Issue by willnx** [willnx/vlab#24]
A user asked this question on a closed issue. Great question! Totally worth it's own issue!

_Hi, I have been trying to create a solid lab setup to work with for customer recreates. I am puzzled as to why half of my machines show connectable-true and the other half are connectable=false._
```
C:\Users\harpew>vlab status
Collecting information about your inventory
Processing inventory records

Username: harpew
Gateway : 10.241.80.xxx

Machines:

Name | IPs | Connectable | Type | Version | Powered | Console
------------+---------------------------+---------------+-----------+-----------+-----------+------------------------------------------------------------------
BH10 | 192.168.1.164 | False | Windows | 10 | On |
BH8007-1 | 192.168.1.71 | True | OneFS | 8.0.0.7 | On |
BH8007-2 | 192.168.1.72 | True | OneFS | 8.0.0.7 | On |
BH8007-3 | 192.168.1.73 | True | OneFS | 8.0.0.7 | On |
BH8120-1 | 192.168.1.21 | False | OneFS | 8.1.2.0 | On |
BH8120-2 | 192.168.1.22 | False | OneFS | 8.1.2.0 | On |
BH8120-3 | 192.168.1.23 | False | OneFS | 8.1.2.0 | On |
harpew2012 | fe80::8808:5fc8:250d:b93c | True | WinServer | 2012R2 | On |
| 192.168.1.163 | | | | |
```

**Comment by willnx** [comment]
### TL;DR
* The `connectable` column is `True` when the defaultGateway in your lab can ping the IP of the specific device. 

* Some servers in vLab have [ICMP Echo/Reply](http://www.firewall.cx/networking-topics/protocols/icmp-protocol/152-icmp-echo-ping.html) disabled by default. 

* If the OneFS [autoconfig](https://github.com/willnx/vlab_onefs/blob/master/vlab_onefs_api/lib/worker/setup_onefs.py) logic fails to make a cluster, there won't be anything to respond to the pings (so try deleting/remaking it).


### Longer explaination
The `connectable` column is the result of a service that runs on the [defaultGateway](https://github.com/willnx/vlab_ipam_api) within your specific lab.

To save me from trying to manage public IP-address space for hundreds of users within a giant network, everyone's lab is behind a [NAT Firewall](https://github.com/willnx/vlab_docs/blob/master/docs/source/overview/networking/network_diagram.png) (the little _wall-thingy_ in that "awesome" [actually terrible] digram I made 😉).

The beta version of vLab had people rely on a [jump server](https://en.wikipedia.org/wiki/Jump_server); so to access your lab, you would have to RPD/SSH into something I named [jumpbox](https://github.com/willnx/vlab_jumpbox). From that jumpbox, you could access the different components in your lab. I really hated this aspect of the vLab beta implementation, and decided to do dynamic [port forwarding](https://en.wikipedia.org/wiki/Port_forwarding). 

As a favor to _future Nick_, I figured it would be really handy to verify that the port forwarding rules have _something on the other end._ So I came up a basic [worker](https://github.com/willnx/vlab_ipam_api/blob/master/vlab_ipam_api/worker.py) service that runs on the defaultGateway. So on some regular interval, that worker checks the IPs assocated with a port forward rule by sending some pings.

When the [vLab CLI](https://github.com/willnx/vlab_cli) executes the `vlab status` logic, [at this point](https://github.com/willnx/vlab_cli/blob/master/vlab_cli/subcommands/status.py#L35), the CLI calls the API on the defaultGatway to get that _connectable_ info about the IPs associated with those port forwarding rules.

So, if the defaultGateway can ping the IP it has a port forwarding rule for, it'll be `True`, and `False` when the pings fail. If a machine is _in the middle_ of being deployed, it'll most likely say `initializing`.
